Scripts/Source/UnforgivingDevicesMain.psc

```diff
@@ -132,6 +132,9 @@ bool Property SlaveTatsInstalled        = false auto
 bool Property OrdinatorInstalled        = false auto
 bool Property ZadExpressionSystemInstalled = false auto
 Bool Property DeviousStrikeInstalled    = False auto
+Bool Property ForHimInstalled           = False auto
+
+Bool Property AllowMenBondage           = False auto
```

Comment:
IMHO, I think that this should be by default **True**. If someone have For Him installed, it should already be taken as that they want to use the bondage on males. The more rare case is  if someone have For Him installed and don't want bondage on males (in which case, they should use the MCM setting, not the other way around)

Reply:
Current men support even with For Him installed is quite so-so. This is why I opted to set it to Disable.
Also set to False is much easier to implement and more robust, as it handles properly all enabling and disabling esps.

For example, I have For Him enabled, but I prefer to play with this disabled, much better results. And UD's for men support is essentially non-existent.

Reply:
But this isn't only about Abadon quest, but about devices in general. ActorIsValidForUD is also used when locking devices, so no devices can be locked on actor if it's false. 
And you are missing one important point, if you had For Him installed when using DD, you knew that it meant that the devices will work for men. So why should it now be reversed ? People will start to flood the support thread asking why For Him is suddenly not working, and you would have to explain to them that they have to actually enable it. So please, change it, so it is by default true. As I said before, the number of people using For Him, and not wanting bondage on man is much smaller than the number of people using For Him, so they can use devices on males.

Reply:
Got your point, will change it.